src/appleseed.studio/mainwindow/rendering/renderingmanager.cpp

```diff
@@ -217,7 +227,8 @@ void RenderingManager::start_rendering(
 
 bool RenderingManager::is_rendering() const
 {
-    return m_master_renderer.get() != nullptr;
+    return m_master_renderer.get() != nullptr
```

Comment:
This is going to be problematic.

When a render is paused, we're still rendering in the sense that a lot of operations (such as editing the scene) are still forbidden.

This probably needs to be reverted.